.github/workflows/ci.yml

```diff
@@ -109,6 +109,42 @@ jobs:
     - name: Check rustdoc links
       run: RUSTDOCFLAGS="--deny broken_intra_doc_links" cargo doc --verbose --workspace --no-deps --document-private-items
 
+  check-clippy:
```

Comment:
@TriplEight does this look sane to you? Did I get the `actions/cache@v2` order right? Your feedback would be very much appreciated.